sale_order_lot_selection/models/stock.py

```diff
@@ -30,7 +30,6 @@ def _prepare_move_line_vals(self, quantity=None, reserved_quant=None):
         vals = super(StockMove, self)._prepare_move_line_vals(
             quantity=quantity, reserved_quant=reserved_quant
         )
-        lot = self.sale_line_id.lot_id
-        if reserved_quant and lot:
-            vals["lot_id"] = lot.id
+        if reserved_quant and self.sale_line_id:
```

Comment:
You no longer check if `self.sale_line_id.lot_id` has a value. Can you explain please?

Reply:
Original suggestion came from @acsonefho so he can give us more details. @acsonefho Would you please share your experience for your suggested code? IMO both code are fine.

Reply:
> Original suggestion came from @acsonefho so he can give us more details. @acsonefho Would you please share your experience for your suggested code? IMO both code are fine.

Sorry but I didn't ask this change. It was already the case into the previous migration request.
https://github.com/OCA/sale-workflow/pull/1040/files#diff-e49e131f8ad91838e08737e015b71376R33

Reply:
@acsonefho I'm sorry. You are right. I have reviewed again and it came from @ThomasBinsfeld

Reply:
@ThomasBinsfeld Any update or should we revert back with original version of code?

Reply:
I asked to make sure that it only applies for sales. I looks ok to me.

https://github.com/OCA/sale-workflow/pull/1040#discussion_r377490022

Reply:
I agree @ThomasBinsfeld Both code will only applies to SO. Thanks for your approval.  @dreispt Do want to revert back original code version or should we move with this version?